grails-app/i18n/messages.properties

```diff
@@ -2947,6 +2947,14 @@ react.invoice.createdBy.label=Created by
 react.invoice.viewDetails.label=View Invoice Details
 react.invoice.edit.label=Edit Invoice
 react.invoice.error.fetching.label=Unable to fetch invoices
+react.invoice.vendor.label=Vendor
+react.invoice.searchField.placeholder.label=Search by invoice number...
+react.invoice.columns.itemCount.label=# items
```

Comment:
In other massages there are `column` instead of `columns` (singular instead of plural) can we keep it consistent?